### PR TITLE
fix: mounting bug

### PR DESCRIPTION
**Pull request by DnMllr**
Mounting order bug

### CODE REVIEW

File: dom-renderers/DOMRenderer.js [hunk 1]
```diff
@@ -449,36 +450,29 @@ DOMRenderer.prototype.resolveChildren = function resolveChildren (element, paren
...
-            );
+    if (this._parent.void)
+        throw new Error(
+            this._parent.path + ' is a void element. ' +
+            'Void elements are not allowed to have children.'
+        );
 
-        if (this._target) {
-            this._parent.element.removeChild(this._target.element);
-            this._removeElCallbackStore.trigger(this._path, this._target);
```
Comment (alexanderGugel) [inline dom-renderers/DOMRenderer.js]: Lost line.

**Comment by alexanderGugel** [comment]
How can this bug be reproduced? Does the change correctly account for changed tag names?

**Comment by alexanderGugel** [comment]
This broke changing tag names and lifecycle events.

**Comment by alexanderGugel** [comment]
@DnMllr How can the bug this was supposed to address be reproduced?

**Comment by jd-carroll** [comment]
This commit is wrong and should be reverted

**Pull request merged**